packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -660,20 +660,13 @@ const verifyBuildCfTypesScript = async (
 	const outputFileContentPost = readFile(join(projectPath, outputFile));
 	const outputFileContentPostLines = outputFileContentPost.split("\n");
 
+	// the file doesn't contain the "Generated by Wrangler" comment anymore
+	expect(outputFileContentPostLines).not.toContain("// Generated by Wrangler");
+
 	// the file still contains the env interface
 	expect(outputFileContentPostLines).toContain(
 		`interface ${envInterfaceName} {`,
 	);
-
-	// the file doesn't contain the "Generated by Wrangler" comment without a timestamp anymore
-	expect(outputFileContentPostLines).not.toContain("// Generated by Wrangler");
-
-	// but it contains the "Generated by Wrangler" comment now with a timestamp
-	expect(
-		/\/\/ Generated by Wrangler on [a-zA-Z]*? [a-zA-Z]*? \d{2} \d{4} \d{2}:\d{2}:\d{2}/.test(
-			outputFileContentPost,
-		),
-	).toBeTruthy();
```

Comment:
The timestamp is no longer included in the generated type files. We should simply assert whether the original comment is removed instead of the new content in the generated files.